contracts/PodSponsorship.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity ^0.6.4;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/Initializable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/utils/ReentrancyGuard.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC777/ERC777.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/introspection/IERC1820Registry.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol";
+import "@opengsn/gsn/contracts/BaseRelayRecipient.sol";
+
+import "@pooltogether/fixed-point/contracts/FixedPoint.sol";
+import "@pooltogether/pooltogether-contracts/contracts/external/openzeppelin/ProxyFactory.sol";
+import "@pooltogether/pooltogether-contracts/contracts/modules/yield-service/YieldServiceInterface.sol";
+import "@pooltogether/pooltogether-contracts/contracts/modules/ticket/Ticket.sol";
+import "@pooltogether/pooltogether-contracts/contracts/modules/timelock/Timelock.sol";
+import "@pooltogether/pooltogether-contracts/contracts/Constants.sol";
+
+
+contract PodSponsorship is Initializable, ReentrancyGuardUpgradeSafe, ERC777UpgradeSafe, BaseRelayRecipient {
+    using SafeMath for uint256;
+
+    event SponsorshipMinted(address indexed pod, address indexed to, uint256 amount);
+    event SponsorshipBurned(address indexed pod, address indexed from, uint256 amount);
```

Comment:
We can ✂️ these.  Don't think they're used here, and we already have Mint and Burn events on the ERC777.